ouroboros-network/test/Test/Socket.hs

```diff
@@ -34,6 +35,12 @@ import           Ouroboros.Network.Serialise
 import           Test.ChainGenerators (TestBlockChainAndUpdates (..))
 import qualified Test.Mux as Mxt
 
+{-
+ - The travis build hosts does not support IPv6 so those test cases are hidden
+ - behind the OUROBOROS_NETWORK_IPV6 define for now.
+ -}
+-- #define OUROBOROS_NETWORK_IPV6
```

Comment:
It might be good to add a cabal flag for this, then running the test would be as simple as `cabal new-run -f ipv6 pkg:ouroboros-network:tests`

Reply:
👍 Will add.